### PR TITLE
feat(web-styles): set up storybook docs

### PR DESCRIPTION
**Pull request by gfellerph**
Adds a storybook project based on the plain HTML preset. Added .scss webpack loaders for easier styles imports.

### CODE REVIEW

File: packages/web-styles/.gitignore [hunk 1]
```diff
@@ -3,7 +3,6 @@ node_modules/
 out-tsc/
 debug.log
 npm-debug.log
-src/**/*.js
```
Comment (oliverschuerch) [inline packages/web-styles/.gitignore:6]: if .js files are now allowed in the entire /web-styles/src directory, the exceptions below become obsolete
- [ ] check if statments below (beginning with `!src/...` become obsolete

File: packages/web-styles/.storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "stories": [
+    "../**/*.stories.@(js|mdx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    {
+      name: "@storybook/preset-scss",
+      opetions: {
```
Comment (oliverschuerch) [inline packages/web-styles/.storybook/main.js:12]: - [ ] typo: property-name should be "options"

File: packages/web-styles/.storybook/main.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        sassLoaderOptions: {
```
Comment (oliverschuerch) [inline packages/web-styles/.storybook/main.js:13]: - [ ] typo: property-name should be "cssLoaderOptions" (not sure, but check https://www.npmjs.com/package/@storybook/preset-scss)

File: packages/web-styles/src/components/button.stories.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+const AnimatedTemplate = (args) => `<button class="btn btn-primary btn-animated"><span>${args.label}</span></button>`;
+
+export const Primary = Template.bind({});
+Primary.args = {
+  label: 'Primary button'
+}
+
+export const PrimaryAnimated = AnimatedTemplate.bind({});
+PrimaryAnimated.args = {
+  label: 'Primary animated'
```
Comment (oliverschuerch) [inline packages/web-styles/src/components/button.stories.js:17]:
```suggestion
  label: 'Primary button animated'
```

File: packages/web-styles/package.json [hunk 5]
```diff
@@ -19,14 +19,30 @@
     "format": "prettier src/**/*.scss --write",
     "test": "jest",
     "test:ci": "jest",
-    "start": "gulp watch"
+    "start": "npm run storybook",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```
Comment (oliverschuerch) [inline packages/web-styles/package.json:24]:
```suggestion
    "build:storybook": "build-storybook"
```

File: packages/web-styles/package.json [hunk 6]
```diff
@@ -35,14 +51,20 @@
...
+    "react": "16",
+    "react-dom": "16",
     "rimraf": "^2.6.3",
-    "sass": "^1.49.7",
+    "sass": "^1.52.3",
+    "sass-loader": "^10",
+    "style-loader": "^2",
     "stylelint": "^14.3.0",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-property-sort-order-smacss": "^8.0.0",
```
Comment (oliverschuerch) [inline packages/web-styles/package.json:62]: merge main branch into pull-request first!

**Review by oliverschuerch — APPROVED** [review]
Let other people review this PR

File: packages/web-styles/.storybook/main.js (hunk 3 above)
Comment (gfellerph) [reply packages/web-styles/.storybook/main.js:13]: There are three loaders used by webpack for loading styles: style-loader, css-loader and sass-loader. Each loader can be passed options so the key is correct in this case.

**Pull request merged**